phpstan.neon.dist

```diff
@@ -15,8 +15,10 @@ parameters:
 		reportUnusedCatchesOfUncheckedExceptions: true
 		uncheckedExceptions:
 			- LogicException
+			- ReflectionException
```

Comment:
Why mark ReflectionException as unchecked?

Reply:
`TestCase::createMock` in phpunit >= v8.0.0 has `@throws RuntimeException`. So line https://github.com/pepakriz/phpstan-exception-rules/blob/master/tests/src/RuleTestCase.php#L87 leads to error `Missing @throws ReflectionException annotation`. But you're right, I'll make an `DynamicMethodThrowTypeExtension` for `TestCase::createMock` method which will throw `RuntimeException` only when class is not autoloaded.